core/domain/collection_services.py

```diff
@@ -785,7 +785,7 @@ def _create_collection(committer_id, collection, commit_message, commit_cmds):
     )
     model.commit(committer_id, commit_message, commit_cmds)
     collection.version += 1
-    create_collection_summary(collection.id, committer_id)
+    regenerate_collection_summary(collection.id, committer_id)
```

Comment:
Over here we aren't regenerating an existing collection summary, right?

Reply:
No but I think the naming is okay, basically both generate and regenerate would be the same functions.